conf/application.conf

```diff
@@ -154,6 +154,7 @@ urls {
   penalties-frontend = "http://localhost:9180/penalties"
   vatOverview = "http://localhost:9152/vat-through-software/vat-overview"
   feedbackUrl = "http://localhost:9514/feedback/penalties-appeals-frontend"
+  whatYouOwe = "http://localhost:9152/vat-through-software/what-you-owe"
```

Comment:
Could you raise an app-config-qa, app-config-staging and app-config-production PR for this?